compiler/rustc_passes/src/dead.rs

```diff
@@ -829,24 +909,37 @@ fn check_mod_deathness(tcx: TyCtxt<'_>, module: LocalDefId) {
                     continue;
                 }
 
+                let mut is_positional = false;
```

Comment:
Positional fields are all or nothing.  Can `is_positional` be computed only once, and passed around?